Optimization/Validation/runValidation.m

```diff
@@ -0,0 +1,145 @@
+function [Results] = runValidation(Misc,DatStore,Mesh,trial,output,optionssol,Results,NMuscles,lMo_scaling_param_opt,lTs_scaling_param_opt,kT_scaling_param_opt)
```

Comment:
Is it possible to include this function in the main script again. Now it is impossible to review. I understand that it might be useful to have a function to build the OCP for the three cases. but not a separate function for the three problems. Because now you just copy pasted this into a functions (without saving lines or code because the functions are only used once....).

This could be useful if this function is very generic and can be used in other project. But with the very specific input structure for this project, I really think this is not useful